src/components/form/Obs.jsx

```diff
@@ -64,6 +64,8 @@ const Obs = (props) => {
         return (
           <Field
             component={Dropdown}
+            field={props.field}
+            handleSelect={props.handleSelect}
```

Comment:
Not sure why changes are required to the Dropdown to allow the checkbox to submit certain values?

Reply:
Hi Mark,
The checkbox needs a way to get the values and that is where we use the dropdown. when the user selects the checkbox the dropdown is displayed and the user can select any option. the dropdown has a custom handleSelect handler, and each time it’s toggled it set’s the value of the checkbox.  The dropdown and the checkbox can still be used separately and as a standalone inputs, no existing functionality was changed.